github/updater.go

```diff
@@ -0,0 +1,98 @@
+package github
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/packwiz/packwiz/core"
+)
+
+type ghUpdateData struct {
+	ModID            string `mapstructure:"mod-id"` // The slug of the repo but named modId for consistency reasons
+	InstalledVersion string `mapstructure:"version"`
```

Comment:
installed version would be v1.0.1 for example makes sense than tag?

Reply:
It might be the same as the version number used by a mod, but it's not very specific... `tag` is very specifically the Git tag the release was created from, whereas a `version` could be the version in the release name, it could be the version in the downloaded file, it could be the commit the tag points to and which the release was created from.